CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+# Kubernetes in Rust : The contributing guide.
+
+Thank you for your interest in this project. In order to contribute to this project, please get familiar with the contributing rules and the git workflow.
+
+## Git workflow :
+
+![Git workflow](https://i.imgur.com/WMNLhY8.jpg)
+
+This project is organized within 2 repositories :
+- [The main repository](https://github.com/dev-sys-do/kudo)
+- [The development repository](https://github.com/do3-2021/kudo)
+
+The development repository is a fork of the main repository. All changes made to this repository are eventually going to get through a pull request of the upstream ( main ) repository.
+
+In the development repository, you will find 3 types of branches :
+- The main branch
+- Team's branch
+- Features ( or bugfix ) branches
+
+You must code in a feature branch. This feature branch will eventually end up in a PR to your team's branch. At this level, only your team's members are required to review. This PR is annotated with **PR 1** in the image above.
+
+After accepting multiples PRs by your team's members, your team's branch will get to a point where you feel it's ready to close an issue ( more on that later ). At this point, you must open a PR from you team's branch to the main branch ( **PR 2** ). This PR must be reviewed by at least 5 reviewers before getting marked as accepted.
+
+When all the teams are satisfied with the work in the main branch, and everybody agrees on the correctness of what's been done, a new PR, from this fork to the parent repo ( the main repo ) ( **PR 3** ), will be opened to be discussed with the parent's repo maintainer. 
+
```

Comment:
So all the above section describes the **internal** workflow that the `do3-2021`class decided to adopt. It's a great workflow, but it's how you're planning to push contributions upstream from a particular fork, the `do3-2021`one. This specific workflow should be documented in your fork, not here.

The `CONTRIBUTING.md`file that you're trying to push through this PR will be the upstream contributing guidelines and you should redacted it accordingly. You can't expect contributors to neither want or be able to follow an internal workflow that you put in place as part of a sys programming course assignment.

Think about `kudo`as a public and open source project that you have to maintain and grow by increasing your contributors pool. Be welcoming to external contributors, ask them to follow a *few* rules that are really important to you, and be flexible with the rest. Make the entry ticket for joining your project low, and guide contributors.